examples/unstructured_2d_dgsem/elixir_shallow_water_convergence.jl

```diff
@@ -0,0 +1,69 @@
+
+using Downloads: download
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the shallow water equations with a particular
+# bottom topography function
+
+@inline function mms_bottom_topography(x, equations::ShallowWaterEquations2D)
+  # periodic bottom for the periodic with a twist mesh domain [0, sqrt(2)]^2
+  return 2.0 + 0.5 * sin(sqrt(2.0)*pi*x[1]) + 0.5 * sin(sqrt(2.0)*pi*x[2])
+end
+
+equations = ShallowWaterEquations2D(9.81, bottom_topography=mms_bottom_topography)
+
+initial_condition = initial_condition_convergence_test
+
+###############################################################################
+# Get the DG approximation space
+
+volume_flux = (flux_wintermeyer_etal, flux_nonconservative_shallow_water_volume)
```

Comment:
The nonconservative terms are made specifically for this volume flux. Thus, it should be something like `flux_wintermeyer_etal, flux_nonconservative_wintermeyer_etal`.